### DISCUSSION TITLE
How to make tailwind site/components copy-protected?

### DISCUSSION THREAD

**Discussion by Musharofchy** [tailwindlabs/tailwindcss#5915] [Help]
Hey, 

Is there any tool or procedure to follow for making tailwind site copy-protected after build? Maybe bulk replace tailwind class names with some random classes but style will remain same or turn into plain HTML.

Please share your thoughts 🙏

**Comment by BenHerbst** [comment]
I am interested into this!

**Comment by mohidmeer** [comment]
This could be great, I have seen the Flobite Tailwind UI liabrary implements something similar like that on there main site
